app/views/posts/_post.html.slim

```diff
@@ -0,0 +1,27 @@
+- @ticket_id = "ticket-#{post.id}"
+- @id = random_dom_id
```

Comment:
when you have more than one line, you can use the `ruby:` filter (at least you can with HAML, I assume you can do it with SLIM too):
```
ruby:
  @ticket_id = 
  @id = 
```